src/gui/progressbarpainter.h

```diff
@@ -42,4 +43,5 @@ class ProgressBarPainter
 private:
     // for painting progressbar with stylesheet option, a dummy progress bar is required
     QProgressBar m_dummyProgressBar;
+    m_dummyProgressBar->setFormat(tr("%p%"));
```

Comment:
It looks like you don't know anything about C++...

Reply:
> That's correct, going blind here. On the other hand, your review is not helpful either.

I don't know what kind of "helpful" review you were hoping for, offering a deliberately invalid patch. 
Did you just want to act as some kind of catalyst, so that someone else would finish this job? You could have stated it initially.

Reply:
Really this code won't compile. You could add this line in constructor of `ProgressBarPainter` class.

Reply:
Okay, I'll update the whole thing once I get to compile the application first. Currently I am having trouble with the libtorrent-rasterbar lib.

Reply:
Also I wonder if it won't confuse translators to see "%p%" without some explanation?